static/panes/compiler.js

```diff
@@ -1231,12 +1290,13 @@ Compiler.prototype.updateButtons = function () {
 };
 
 Compiler.prototype.handlePopularArgumentsResult = function (result) {
-    var popularArgumentsMenu = this.domRoot.find('div.populararguments div.dropdown-menu');
-    popularArgumentsMenu.html('');
+    var popularArgumentsMenu = $(this.domRoot.find('div.populararguments div.dropdown-menu'));
 
-    if (result) {
-        var addedOption = false;
+    while(popularArgumentsMenu && popularArgumentsMenu.children().length > 1) {
```

Comment:
why the `popularArgumentsMenu &&` here? I don't think it can ever be false-y (else the code below is going to break too)

Reply:
Removed